pkg/minikube/cluster/cluster.go

```diff
@@ -386,9 +386,12 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	}
 
 	if config.ShellProxyEnv != "" {
+		print("-- Setting proxy information ... ")
```

Comment:
here is a print left. Given our latest experiences and for consistency sake, I recommend to use fmt